android/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.2-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.5-all.zip
```

Comment:
Was this supposed to be 
distributionUrl=https\://services.gradle.org/distributions/gradle-6.7-all.zip
?

Reply:
Yes it was supposed!